src/main/scala/mesosphere/marathon/state/PortDefinition.scala

```diff
@@ -14,7 +14,7 @@ case class PortDefinition(
 
 object PortDefinition {
   implicit val portDefinitionValidator = validator[PortDefinition] { portDefinition =>
-    portDefinition.protocol is oneOf(DiscoveryInfo.Port.AllowedProtocols)
+    portDefinition.protocol.split(',').toIterable is every(oneOf(DiscoveryInfo.Port.AllowedProtocols))
```

Comment:
@jasongilanfarr should we update endpoints to define `protocol` as an array? or do the same thing that we're apparently doing for apps; parsing a string